payments-core/api/payments-core.api

```diff
@@ -5047,6 +5047,13 @@ public abstract class com/stripe/android/networking/StripeRequest {
 	protected final fun getQuery ()Ljava/lang/String;
 }
 
+public final class com/stripe/android/payments/PaymentFlowFailureMessageFactory {
```

Comment:
Take a look at the comments in these two commits on items that were opened in order to split payment sheet.  I imagine some of them can now be made internal:
https://github.com/stripe/stripe-android/pull/4120/commits/6c0a99330a322729e01c3a902681fc4176d3ea2e
https://github.com/stripe/stripe-android/pull/4120/commits/c9dc6aa4a80c55f36396755f1454c6b42ea0d4dc

Reply:
Looking through them now. It'll take me a bit to go through them all :)

Reply:
As discussed offline, we can comb through the classes to convert back to internal after a similar replacement in [DefaultFlowController](https://github.com/stripe/stripe-android/blob/8e00cc01f2899fbc1b81eb98ba7874e2b922edf7/paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt#L76) is done

Reply:
converted back to internal [here](https://github.com/stripe/stripe-android/pull/4159/commits/51992c5164de08c278682c50262c5d5244869f1f)

Will come back to this in a new PR